### PR TITLE
Reflecting event changes

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: src/master/core/events.py [hunk 1]
```diff
@@ -149,39 +183,102 @@ def data(self):
...
+                                   'value': Temperature.system_value_to_temperature(self._data[1])}]
             elif self._action == 1:
-                sensor_type = Event.SensorType.HUMIDITY
-                sensor_value = Humidity.system_value_to_humidity(self._data[1])
+                sensor_values += [{'type': Event.SensorType.HUMIDITY,
+                                   'value': Humidity.system_value_to_humidity(self._data[1])}]
             elif self._action == 2:
-                sensor_type = Event.SensorType.BRIGHTNESS
-                sensor_value = self._word_decode(self._data[0:2])
+                sensor_values += [{'tye': Event.SensorType.BRIGHTNESS,
```
Comment (LnL7) [inline src/master/core/events.py:212]:
```suggestion
                sensor_values += [{'type': Event.SensorType.BRIGHTNESS,
```

File: src/master/core/events.py [hunk 2]
```diff
@@ -149,39 +183,102 @@ def data(self):
...
+                                   'value': self._word_decode(self._data[0:2])}]
+            elif self._action == 3:
+                sensor_values += [{'type': Event.SensorType.CO2,
```
Comment (LnL7) [inline src/master/core/events.py:215]: FYI while events for new sensor types are handled here the sensor controller still won't sync the new types. Some changes are still needed there to fully support co2/voc sensors.

File: src/master/core/events.py (hunk 1 above)
Comment (khenderick) [reply src/master/core/events.py:212]: Nice catch

File: src/master/core/events.py (hunk 2 above)
Comment (khenderick) [reply src/master/core/events.py:215]: Good point, I've made sure the SensorController wouldn't raise exceptions if these new sensor types would be send.

**Review by LnL7 — APPROVED** [review]

**Pull request merged**